### PR TITLE
added deno code for invoice generator

### PR DESCRIPTION
**Pull request by camperjett**
Signed-off-by: Jaideep Sharma [sharmajaideep1906@gmail.com](mailto:sharmajaideep1906@gmail.com)

Closes: https://github.com/appwrite/appwrite/issues/4089

PS: This PR is submitted for Hacktoberfest'22

### Screenshots:
Screenshot from 2022-10-05 17-09-44
Screenshot from 2022-10-05 18-25-43
Screenshot from 2022-10-05 18-26-08

### CODE REVIEW

**Comment by PineappleIOnic** [comment]
Hey There 👋
Thanks for the PR! Could you please attach an image to this issue of the function being successfully executed?

**Comment by camperjett** [comment]
@PineappleIOnic Check it out!

**Comment by camperjett** [comment]
PTAL 👀 @PineappleIOnic

File: deno/generate_invoice/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts --rm --interactive --tty --volume $PWD:/usr/code openruntimes/deno:1.14 sh /usr/local/src/build.sh
+```
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts -e INTERNAL_RUNTIME_KEY=secret-key -e INTERNAL_RUNTIME_ENTRYPOINT=mod.ts --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/deno:1.14 sh /usr/local/src/start.sh
+```
+
+Your function is now listening on port `3000`, and you can execute it by sending `POST` request with appropriate authorization headers. To learn more about runtime, you can visit Deno runtime [README](https://github.com/open-runtimes/open-runtimes/tree/main/runtimes/deno-1.14).
```
Comment (stnguyen90) [inline deno/generate_invoice/README.md:36]: Please try to use the newest image: `openruntimes/deno:v2-1.24`

File: .gitignore [hunk 2]
```diff
@@ -25,4 +25,6 @@ code.tar.gz
 .Spotlight-V100
 .Trashes
 ehthumbs.db
-Thumbs.db
\ No newline at end of file
+Thumbs.db
+
+.vscode
```
Comment (stnguyen90) [inline .gitignore:30]: no need to add this. we may want to add .vscode in the future.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by christyjacob4** [comment]
@camperjett please address the comments

**Comment by Meldiron** [comment]
Hey there 👋
We are temporarily closing your PR as there has been no activity on it. Feel free to re-open it when you're ready to start working on it again 😇

**Pull request closed without merging**